endpointmanager/pkg/endpointmanager/fhirendpoint_test.go

```diff
@@ -175,63 +190,3 @@ func Test_FHIREndpointEqual(t *testing.T) {
 		t.Errorf("Nil endpoint 1 should equal nil endpoint 2.")
 	}
 }
-
-func Test_AddOrganizationName(t *testing.T) {
```

Comment:
Why were these tests removed?

Reply:
This function is no longer used since we don't need to add organizations to the existing organization list so that the entire array with the new entries can be added to the database. Now, new organizations are added by checking to see if an entry exists in the db, and if not, it adds the new entry, otherwise it just updates the update time. Therefore, since this function is no longer used and was removed, the tests were removed!